FAQ.md

```diff
@@ -0,0 +1,104 @@
+# User FAQ
+
+This document consists of frequently asked questions intended for users.
+
+# Important Terminology
+## Machine vs Node vs VM/Instance
+Machine means a “machine object” in the OpenShift API.  This object is reconciled by the “machine-controller” and serves as a record of information for cloud resources.
+
+“VM” or “Instance” means the actual compute resource running on the cloud.
+
+“Node” means the kubelet and the “node” object in the API.
+
+These terms, while obviously related, are not interchangeable.  Oftentimes, people will say “I deleted the node” and what they really mean is “I deleted the VM using the cloud provider’s CLI tool.”  As you can see, this can cause confusion and delay in resolving your issue.
+
+# Machines
+
+## Machine Phases?
+Please see docs for machine phases: <link goes here>
```

Comment:
Do we have docs for this?

Reply:
https://github.com/openshift/enhancements/blob/master/enhancements/machine-api/machine-instance-lifecycle.md